lib/index.js

```diff
@@ -13,47 +12,32 @@ module.exports = function modelBase (bookshelf, params) {
     constructor: function () {
       bookshelfModel.apply(this, arguments)
 
-      if (this.validate) {
-        var baseValidation = {
-          // id might be number or string, for optimization
-          id: Joi.any().optional(),
-          created_at: Joi.date().optional(),
-          updated_at: Joi.date().optional()
-        }
+      var Model = bookshelf.model(this.modelName);
+
+      Checkit.Validator.prototype.uniqueness = function(val, field, scope) {
```

Comment:
Instead of this, let's inject the model via `params` key in the Checkit validator.